vspace/src/main/java/edu/asu/diging/vspace/web/staff/UpdateModuleStatusController.java

```diff
@@ -0,0 +1,33 @@
+package edu.asu.diging.vspace.web.staff;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.impl.ModuleStatus;
+import edu.asu.diging.vspace.core.services.IModuleManager;
+
+@Controller
+public class UpdateModuleStatusController {  
+    @Autowired
+    private IModuleManager moduleManager;
+    
+    @RequestMapping(value="/staff/module/{moduleId}/status", method=RequestMethod.POST)
+    public String updateStatus(HttpServletRequest request,RedirectAttributes attributes, @PathVariable("moduleId") String spaceId, @RequestParam("statusParam") ModuleStatus status) {
+        IModule module = moduleManager.getModule(spaceId);
+        module.setModuleStatus(status);
+        moduleManager.storeModule(module);
+        attributes.addAttribute("alertType", "success");
+        attributes.addAttribute("message", "Status successfully updated!");
+        attributes.addAttribute("showAlert", "true");
```

Comment:
Does this work? doesn't it have to be `addFlashAttribute`?

Reply:
<img width="1175" alt="Screen Shot 2022-01-20 at 1 00 34 PM" src="https://user-images.githubusercontent.com/22446190/150413054-20343b9c-8bdc-4a00-98ee-1c39440f37e5.png">

The status when updated appears like this